src/namespaces.json

```
@@ -198,5 +198,24 @@
     "min": 0,
     "invalid": [],
     "visible": false
+  },
+  "unidao": {
+    "key": "unidao",
+    "symbol": "MRQ",
+    "name": "UniDAO",
+    "defaultView": "all",
+    "address": "0xc1d204fc521f9e125a55dd8b2152d7344766cc21",
```

The address need to be checksumed
```suggestion
    "address": "0xC1d204fC521F9e125a55dd8b2152d7344766cC21",
```